manifests/docker.pp

```diff
@@ -34,12 +34,19 @@
   # This directory doesn't really belong to us, but we need it to exist and
   # docker apparently doesn't create it. Therefore, we manage it only if
   # nothing else already does.
-  unless defined(File['/run/docker/plugins']) {
-    file { '/run/docker/plugins':
+  unless defined(File['/etc/docker/plugins']) {
+    file { '/etc/docker/plugins':
       ensure => 'directory',
     }
   }
 
+  file { '/etc/docker/plugins/xylem.spec':
```

Comment:
That's what the `xylem.xpec` file does: https://docs.docker.com/engine/extend/plugin_api/#plugin-discovery